lightning/src/chain/chainmonitor.rs

```diff
@@ -203,19 +222,41 @@ where C::Target: chain::Filter,
 	pub fn get_claimable_balances(&self, ignored_channels: &[&ChannelDetails]) -> Vec<Balance> {
 		let mut ret = Vec::new();
 		let monitors = self.monitors.read().unwrap();
-		for (_, monitor) in monitors.iter().filter(|(funding_outpoint, _)| {
+		for (_, monitor_state) in monitors.iter().filter(|(funding_outpoint, _)| {
 			for chan in ignored_channels {
 				if chan.funding_txo.as_ref() == Some(funding_outpoint) {
 					return false;
 				}
 			}
 			true
 		}) {
-			ret.append(&mut monitor.get_claimable_balances());
+			ret.append(&mut monitor_state.monitor.get_claimable_balances());
 		}
 		ret
 	}
 
+	/// Gets the [`LockedChannelMonitor`] for a given funding outpoint, returning an `Err` if no
+	/// such [`ChannelMonitor`] is currently being monitored for.
+	pub fn get_current_monitor(&self, funding_txo: OutPoint) -> Result<LockedChannelMonitor<'_, ChannelSigner>, ()> {
```

Comment:
I'd prefer `s/get_current_monitor/get_monitor`, since aren't all ChannelMonitors (at least ones that have at least 1 revoked state) always monitored for?

Reply:
Gotcha. I'm in favor of similarly renaming `list_current_monitors` then as well